site.conf

```diff
@@ -8,10 +8,15 @@
   regdom = 'DE',
   
   default_domain = 'ffbab',
+  
   config_mode = {
     geo_location = {
       show_altitude = false,
     },
+  remote_login = {
+    show_password_form = true,
+    min_password_length = 10,
```

Comment:
This does not resolve the underling issue.

Reply:
What kind of code changes are you requesting instead?